src/storage/mvcc/reader/scanner/backward.rs

```diff
@@ -26,8 +26,8 @@ const REVERSE_SEEK_BOUND: u64 = 16;
 /// Internally, for each key, rollbacks are ignored and smaller version will be tried. If the
 /// isolation level is SI, locks will be checked first.
 ///
-/// Use `ScannerBuilder` to build `BackwardScanner`.
-pub struct BackwardScanner<S: Snapshot> {
+/// Use `ScannerBuilder` to build `BackwardKvScanner`.
+pub struct BackwardKvScanner<S: Snapshot> {
```

Comment:
`KV` is better?

Reply:
According to Rust's [naming conventions](https://doc.rust-lang.org/1.0.0/style/style/naming/README.html), acronyms count as one word.